modules/phase_field/doc/content/modules/phase_field/Nucleation/DiscreteNucleation.md

```diff
@@ -27,16 +27,78 @@ adaptivity, and a postprocessor:
   that triggers refinement at the point of nucleus insertion.
 - [`DiscreteNucleationTimeStep`](postprocessors/DiscreteNucleationTimeStep.md) - a
   postprocessor to provide a time step limit for new nuclei to use with
-  [IterationAdaptiveDT](/IterationAdaptiveDT.md)
+  [IterationAdaptiveDT](/IterationAdaptiveDT.md).
+- [`DiscreteNucleationData`](postprocessors/DiscreteNucleationData.md) - a
+  postprocessor to provide diagnostic data on nucleation events.
+- [DiscreteNucleationAux](/DiscreteNucleationAux.md) - an auxkernel to map the
+  [`DiscreteNucleationMap`](/DiscreteNucleationMap.md) to an auxvariable field.
```

Comment:
I think it's important to spell out for the Map that it has the central location (I think, correct me if I'm wrong) as well as the shape of the nuclei, and the nuclei have some finite size.

Reply:
This is spelled out in `DiscreteNucleationMap.md`

Reply:
ok.  I didn't click through.